core/storage/trie/on_read.hpp

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef KAGOME_STORAGE_TRIE_ON_READ_HPP
+#define KAGOME_STORAGE_TRIE_ON_READ_HPP
+
+#include <memory>
+#include <unordered_map>
+
+#include "common/blob.hpp"
+#include "common/buffer.hpp"
+
+namespace kagome::storage::trie {
+  struct OnRead {
+    auto onRead() {
+      return [this](const common::Hash256 &hash, common::BufferView raw) {
```

Comment:
Can we give here more meaningful name for params? Like key, value?

Reply:
`hash` and `raw` are correct